XCUITest/PastenGoTest.swift

```diff
@@ -51,33 +51,4 @@ class PastenGoTest: BaseTestCase {
         waitForExistence(app.buttons["Search for mozilla"])
         XCTAssertTrue(app.buttons[UIPasteboard.general.string!].isEnabled)
     }
-
-    // Smoketest
-    // Test Paste & Go feature
-    func testPastenGo() {
```

Comment:
Is it a good idea to completly remove this test? If it's failing, maybe we should disable it for now and change it later. To disable it you can do it like this:
```
func testPastenGo() throw {
	throw XCTSkip("This test needs to be updated or removed")
	...
```

Reply:
Sorry for the confusion, I thought this functionality was removed and I removed the test.